### ISSUE TITLE
[Feature Request] Allow image insertion with FileSafe integration 

### ISSUE THREAD

**Issue by mburnett8** [standardnotes/forum#1265]
Allow images to be inserted in document with FileSafe integration.

<img width="266" alt="image" src="https://user-images.githubusercontent.com/49873044/101391883-f8791e80-3892-11eb-9416-d8abae4d82ed.png">

**Comment by JaspalSuri** [comment]
Thanks for the feedback! We plan to add the ability to natively upload and view images within the apps in a future update. This will allow users to access a list of attached files regardless of which editor they are currently using.

As for changes that will be made to the Plus Editor, with regards to it automatically stripping HTML tags, please see [here](https://github.com/standardnotes/plus-editor/issues/44#issuecomment-797473306). 🙂

**Comment by jmason888** [comment]
That's great news.

I'd been saving images I *want to* use in my Plus notes into FileSafe.... but I didn't have a good way to include them (unless they were public on the web, of course, in which case I just aimed an `<img src="...">` tag at them), so I just flagged them as `[IMG: such-and-such-image goes here]`.  So when this feature goes live I can just search for that `[IMG:` string and start attaching them.
